### ISSUE TITLE
getFilesStream is returning a function instead a ReadableStream

### ISSUE THREAD

**Issue by AnthonyLzq** [googleapis/nodejs-storage#1695]
According to the [docs](https://googleapis.dev/nodejs/storage/latest/Bucket.html#getFilesStream), the `getFilesStream` method should return a `ReadableStream`, but in the IntelliSense of VSCode, the return of the method is a `Function`.

![image](https://user-images.githubusercontent.com/43073718/139968777-5534039b-f40a-4da1-91ed-fedad4879dfc.png)

The code I have is:

```typescript
import { getStorage } from 'firebase-admin/storage'
const bucket = getStorage().bucket()
bucket.getFilesStream()
```

Also, in the `bucket.d.ts` file there is the following:

![image](https://user-images.githubusercontent.com/43073718/139969053-69180018-ac74-4cc8-bef7-99c411cf2d20.png)

#### Environment details

  - OS: Manjaro Linux x86_64
  - Node.js version: 16.13.0
  - npm version: 8.1.0
  - `@google-cloud/storage` version: 5.15.3

#### Steps to reproduce

  1. Copy the provided code and check the return of the method provided from the IntelliSense.

**Comment by ddelgrosso1** [comment]
Thank you for bringing this to our attention @AnthonyLzq. I have confirmed that this does incorrectly show in intellisense (and the types definition file). I will look into fixing the definition and let you know when it is fixed.

**Comment by ddelgrosso1** [comment]
@AnthonyLzq  this should be fixed in `v5.15.5` once it gets merged / released.
